app/services/db/sp_return_log.rb

```diff
@@ -11,7 +12,15 @@ def self.create_return(request_id:, user_id:, billable:, ial:, issuer:, requeste
         returned_at: Time.zone.now,
       )
     rescue ActiveRecord::RecordNotUnique
+      # Can be removed after deploy of RC 185
+      ::SpReturnLog.where(request_id: request_id).update_all(
```

Comment:
I would like to, but I don't think so since we don't want to update the `requested_at` row on the update (but we do want to include it on the insert). I can't find if or where Rails exposes having `DO UPDATE` on a different set of columns.

Reply:
it's possible via raw SQL to just set different columns I think

```sql
INSERT INTO sp_return_logs
  (request_id, user_id, billable, ial, issuer, requested_at, returned_at)
VALUES 
  (%{request_id}, %{user_id}, %{billable}, %{ial}, %{issuer}, %{requested_at}, %{returned_at}
ON CONFLICT (request_id)
DO UPDATE SET
  user_id = %{user_id}, 
  returned_at = %{returned_at}
  billable = %{billable}
  ial = %{ial}
```

Reply:
I'm cool with either way since it'll be deleted after Thursday

Reply:
your call!